woocommerce/woocommerce-admin
# Only show the Jetpack logo in the header if we need to setup Jetpack

Fixes #3146.

This PR updates the header bar on the start/plugin pages, so that the Jetpack logo only shows if we actually need to setup and activate Jetpack.

### Screenshots

<img width="978" alt="Screen Shot 2019-11-01 at 11 55 15 AM" src="https://user-images.githubusercontent.com/689165/68038021-5e15d700-fc9f-11e9-8ce7-5740b4aa3d4b.png">

<img width="978" alt="Screen Shot 2019-11-01 at 11 55 35 AM" src="https://user-images.githubusercontent.com/689165/68038036-6706a880-fc9f-11e9-8be4-b3fd033e7cff.png">

### Detailed test instructions:

- Disable just WooCommerce Services, and go to the first page of the profiler.
- Note that only the WooCommerce logo is shown in the header.
- Disable or disconnect Jetpack.
- Note that both logos are shown in the header.
